lib/routes.jsx

```diff
@@ -152,10 +157,19 @@ function buildRoutes() {
     //Add each locale's routes to the array of urls that the server uses for route matching
     urls.forEach(function(key) {
       var newkey = locale + "/" + key;
-      
+
       localeURLs.push(newkey);
     });
+  });
 
+  MAKER_PARTY_LOCALES.forEach((locale) => {
+    localeURLs.push(`${locale}`, `${locale}/events`);
+    routes.push(
+      <Route key={locale} path={locale} component={require('../components/page.jsx')}>
+        <IndexRedirect to="/" />
```

Comment:
I think I was wondering if we should redirect `/de/` to `/de/events/`

I'm fine if we don't it's just a thought.